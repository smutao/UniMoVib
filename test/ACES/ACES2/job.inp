a test job of ACES 2.
Note: The atoms have been reordered in the output file.

 $contrl
   qcprog="aces"
 $end

 $qcdata
   fchk="ch4-a2.out"
 $end
