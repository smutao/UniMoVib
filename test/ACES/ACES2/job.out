 =======================================================================================================================
       _/     _/                _/     _/            _/     _/
      _/     _/            _/  _/|  _/_/            _/     _/  _/  _/        UniMoVib:          A unified interface for
     _/     _/   _/_/_/       _/ |_/ _/   _/_/_/   _/     _/      _/           molecular harmonic vibrational frequency
    _/     _/  _/    _/  _/  _/  _/ _/  _/    _/  _/     _/  _/  _/_/_/        calculations.
   _/     _/  _/    _/  _/  _/     _/  _/    _/  _/    _/   _/  _/   _/
  _/     _/  _/    _/  _/  _/     _/  _/    _/   _/  _/    _/  _/   _/       Ver. 1.0.0, MAY 02, 2017
   _/_/_/   _/    _/  _/  _/     _/   _/_/_/     _/_/     _/  _/_/_/         https://github.com/zorkzou/UniMoVib
 =======================================================================================================================


 QC Program:      ACES
 FCHK file:       ch4-a2.out


 Cartesian coordinates (Angstrom)
 ------------------------------------------------------------------------------------------
   No.   Atom    ZA                 X             Y             Z                Mass
 ------------------------------------------------------------------------------------------
     1    H       1            0.00000000   -0.88418541    0.62521350            1.00782504
     2    C       6            0.00000000    0.00000000    0.00000000           12.00000000
     3    H       1            0.00000000    0.88418541    0.62521350            1.00782504
     4    H       1            0.88418541    0.00000000   -0.62521350            1.00782504
     5    H       1           -0.88418541    0.00000000   -0.62521350            1.00782504
 ------------------------------------------------------------------------------------------


 <<< SYMMETRY >>>
 Molecular Point Group (Z)   :   Td  
 Molecular Point Group (Z+M) :   Td  


 ************************************
 ***  Properties of Normal Modes  ***
 ************************************

 Results of vibrations:
 Normal frequencies (cm**-1), reduced masses (AMU),
 force constants (mDyn/A), IR intensities (km/mol)

                                                   1                                 2                                 3
          Irreps                                  T2                                T2                                T2
     Frequencies                           1520.3234                         1520.3234                         1520.3234
  Reduced masses                              1.1821                            1.1821                            1.1822
 Force constants                              1.6099                            1.6099                            1.6099
  IR intensities                              0.0000                            0.0000                            0.0000
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1   1        -0.60437   0.00354   0.00793      -0.01476  -0.14513  -0.32473       0.00000   0.32482   0.37487
           2   6         0.12589  -0.00307  -0.00000       0.00307   0.12589  -0.00000      -0.00000  -0.00000  -0.12593
           3   1        -0.60437   0.00354  -0.00793      -0.01476  -0.14513   0.32473       0.00000  -0.32482   0.37487
           4   1        -0.14513   0.01476  -0.32473      -0.00354  -0.60437  -0.00793       0.32482   0.00000   0.37487
           5   1        -0.14513   0.01476   0.32473      -0.00354  -0.60437   0.00793      -0.32482   0.00000   0.37487

                                                   4                                 5                                 6
          Irreps                                   E                                 E                                A1
     Frequencies                           1739.7776                         1739.7776                         3186.7592
  Reduced masses                              1.0078                            1.0078                            1.0078
 Force constants                              1.7973                            1.7973                            6.0302
  IR intensities                              0.0000                            0.0000                            0.0000
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1   1         0.49921   0.01625   0.02298      -0.02815   0.28826   0.40757       0.00000  -0.40822   0.28871
           2   6         0.00000   0.00000   0.00000      -0.00000   0.00000  -0.00000       0.00000   0.00000   0.00000
           3   1        -0.49921  -0.01625   0.02298       0.02815  -0.28826   0.40757       0.00000   0.40822   0.28871
           4   1        -0.01625  -0.49921  -0.02298      -0.28826   0.02815  -0.40757       0.40822   0.00000  -0.28871
           5   1         0.01625   0.49921  -0.02298       0.28826  -0.02815  -0.40757      -0.40822  -0.00000  -0.28871

                                                   7                                 8                                 9
          Irreps                                  T2                                T2                                T2
     Frequencies                           3280.1135                         3280.1135                         3280.1135
  Reduced masses                              1.0989                            1.0989                            1.0989
 Force constants                              6.9661                            6.9661                            6.9660
  IR intensities                              0.0000                            0.0000                            0.0000
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1   1        -0.01714   0.40349  -0.29761       0.01740   0.39743  -0.29315      -0.00000   0.41775  -0.27095
           2   6        -0.06388  -0.06485   0.00000       0.06485  -0.06388   0.00000      -0.00000   0.00000   0.09102
           3   1        -0.01714   0.40349   0.29761       0.01740   0.39743   0.29315      -0.00000  -0.41775  -0.27095
           4   1         0.39743  -0.01740  -0.29315      -0.40349  -0.01714   0.29761       0.41775   0.00000  -0.27095
           5   1         0.39743  -0.01740   0.29315      -0.40349  -0.01714  -0.29761      -0.41775   0.00000  -0.27095

 Results of translations and rotations:

                                                   1                                 2                                 3
          Irreps                                  T2                                T2                                T2
     Frequencies                              6.6986                            6.6986                            6.6984
  Reduced masses                              3.2063                            3.2063                            3.2063
 Force constants                              0.0001                            0.0001                            0.0001
  IR intensities                              0.0000                            0.0000                            0.0000
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1   1         0.44721   0.00000   0.00000       0.00000   0.44721   0.00000       0.00000   0.00000   0.44721
           2   6         0.44721   0.00000   0.00000       0.00000   0.44721   0.00000       0.00000   0.00000   0.44721
           3   1         0.44721   0.00000   0.00000       0.00000   0.44721   0.00000       0.00000   0.00000   0.44721
           4   1         0.44721   0.00000   0.00000       0.00000   0.44721   0.00000       0.00000   0.00000   0.44721
           5   1         0.44721   0.00000   0.00000       0.00000   0.44721   0.00000       0.00000   0.00000   0.44721

                                                   4                                 5                                 6
          Irreps                                  T1                                T1                                T1
     Frequencies                             -0.0000                            0.1204                            0.1204
  Reduced masses                              1.0078                            1.0078                            1.0078
 Force constants                             -0.0000                            0.0000                            0.0000
  IR intensities                              0.0000                            0.0000                            0.0000
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1   1         0.50000   0.00000   0.00000      -0.35355   0.00000   0.00000      -0.00000  -0.35355  -0.50000
           2   6        -0.00000   0.00000   0.00000       0.00000   0.00000   0.00000       0.00000   0.00000   0.00000
           3   1        -0.50000   0.00000  -0.00000      -0.35355   0.00000   0.00000      -0.00000  -0.35355   0.50000
           4   1        -0.00000   0.50000  -0.00000       0.35355   0.00000   0.50000       0.00000   0.35355  -0.00000
           5   1        -0.00000  -0.50000   0.00000       0.35355  -0.00000  -0.50000       0.00000   0.35355  -0.00000


 *********************************************
 ***   Thermal Contributions to Energies   ***
 *********************************************

 Temperature               :       298.150000    Kelvin
 Pressure                  :         1.000000    Atm
 Molecular mass            :        16.031300    AMU
 Elec. total energy        :         0.000000    Hartree
 Scale factor of Frequency :         1.000000
 Rot. symmetry number      :        12
 The Td   point group is used to calculate rotational entropy.

 Principal axes and moments of inertia in atomic units:
                                     1                   2                   3
     Eigenvalues --                 11.254596           11.254596           11.254596
           X                         0.000000            0.000000            1.000000
           Y                         0.000000           -1.000000            0.000000
           Z                         1.000000            0.000000            0.000000

 Rotational temperatures             7.695862            7.695862            7.695862    Kelvin
 Rot. constants A, B, C              5.348898            5.348898            5.348898    cm**-1
                                   160.355932          160.355932          160.355932    GHz

 Thermal correction energies                              Hartree            kcal/mol
 Zero-point Energy                          :            0.047996           30.117678
 Thermal correction to Energy               :            0.050845           31.905886
 Thermal correction to Enthalpy             :            0.051789           32.498372
 Thermal correction to Gibbs Free Energy    :            0.030702           19.265609


 Job terminated correctly! Tue May 02 16:00:52 2017
